packages/skills/CLAUDE.md

```
@@ -0,0 +1,212 @@
+2. 遵循命名规范（kebab-case，描述性命名：`http-controller.md`、`mcp-controller.md`）
+3. 包含完整的代码示例和决策树
+4. 更新父级 `SKILL.md` 引用新文档
+5. 如果 `references/` 中已有文件，移除 `.gitkeep`
+
+## Skill 评测
+
+评测用例存放在 `packages/skills/eval/` 目录下，用于验证 AI 使用 skill 后的回答质量。
+
+**评测文件结构：**
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)